app/assets/stylesheets/darkswarm/shop_tabs.scss

```diff
@@ -34,17 +34,17 @@
     >a {
       outline: none;
       display: block;
-      color: $grey-500;
+      color: $black;
       font-family: "Oswald", sans-serif;
     }
 
     a {
       @include headingFont;
 
       background: transparent;
-      text-transform: uppercase;
+      text-transform: capitalize;
```

Comment:
I must be missing something because neither this nor the color change are defined within a media query. Just the font-size below. Therefore, this is what I see with dev tools

![Screenshot from 2020-10-06 11-03-14](https://user-images.githubusercontent.com/762088/95181429-91bc8380-07c3-11eb-8ba1-08af6e235645.png)

Reply:
yes, it's correct. You would have to see the discussion in the issue.

But bottom line: color and capitalize are for all devises, no media query. And then we only make font size smaller for smaller screens :+1: